sdk/python/feast/infra/offline_stores/bigquery.py

```diff
@@ -52,6 +52,9 @@ class BigQueryOfflineStoreConfig(FeastConfigBaseModel):
     project_id: Optional[StrictStr] = None
     """ (optional) GCP project name used for the BigQuery offline store """
 
+    location: Optional[StrictStr] = None
+    """ (optional) GCP location name used for the BigQuery offline store """
```

Comment:
I think it would be really useful to provide an example as part of this comment, like `US` or `EU`

Reply:
Good call.  Added some additional documentation.